narwhals/expr_str.py

```diff
@@ -370,6 +370,13 @@ def tail(self: Self, n: int = 5) -> ExprT:
     def to_datetime(self: Self, format: str | None = None) -> ExprT:  # noqa: A002
         """Convert to Datetime dtype.
 
+        Notes:
+            - defaults to nanosecond time unit, Polars to microsecond.
```

Comment:
I can't work out if dropping `pandas` was intentional?

```suggestion
            - pandas defaults to nanosecond time unit, Polars to microsecond.
```

Reply:
thanks for spotting this!